src/react-components/media-browser.js

```diff
@@ -263,6 +257,32 @@ class MediaBrowser extends Component {
     this.browserDiv.scrollTop = 0;
   };
 
+  handleCopyAvatar = async (e, entry) => {
+    e.preventDefault();
+    await remixAvatar(entry.id, entry.name);
```

Comment:
This fails silently with a 401 response if you are not signed in. In comparison, the "create avatar" action shows you the sign-in dialog.